templates/rails_helper.rb

```diff
@@ -1,6 +1,12 @@
 ENV["RAILS_ENV"] = "test"
 
 require File.expand_path("../../config/environment", __FILE__)
+if ENV["DATABASE_URL"]
+  abort(
+    "A database url has been detected as set. " \
```

Comment:
I'd say here a more concrete "`DATABASE_URL` environmental variable is set.", given it can be transparent to the user.